front/app/hooks/files/useSyncFiles.ts

```diff
@@ -0,0 +1,93 @@
+import { useCallback } from 'react';
+
+import fileAttachmentsKeys from 'api/file_attachments/keys';
+import useAddFileAttachment from 'api/file_attachments/useAddFileAttachment';
+import useDeleteFileAttachment from 'api/file_attachments/useDeleteFileAttachment';
+import useUpdateFileAttachment from 'api/file_attachments/useUpdateFileAttachment';
+
+import { queryClient } from 'utils/cl-react-query/queryClient';
+
+import { SyncFilesArguments } from './types';
+
+export function useSyncFiles() {
+  const { mutateAsync: addFileAttachment } = useAddFileAttachment({
```

Comment:
Replaced the useSyncPhaseFiles with this general one, that I can now re-use for the Events and general Project attachments.